get_hashsets.js

```diff
@@ -0,0 +1,47 @@
+"use strict";
+
+require("dotenv").load();
+
+const fs = require("fs");
+const path = require("path");
+const request = require("request");
+
+const HIBP_API_ROOT = process.env.HIBP_API_ROOT;
+const HIBP_API_TOKEN = process.env.HIBP_API_TOKEN || '';
```

Comment:
Shouldnt these be imported from "./app-constants.js"?
https://github.com/mozilla/blurts-server/blob/c94d8403a234c3714d46c826fd1b02304ce3e31b/email-utils.js#L3